ydb/docs/en/core/reference/ydb-cli/sql.md

```diff
@@ -0,0 +1,120 @@
+# Query execution
+
+You can use the `{{ ydb-cli }} sql` subcommand to execute any SQL query. The query can be of any type (DDL, DML e.t.c.) and can consist of several sub-queries. The `{{ ydb-cli }} sql` subcommand establishes a streaming connection and retrieves data through it. With the in-stream query execution, no limit is imposed on the amount of data read.
```

Comment:
IIRC, there was a technical debt of some kinds of queries that can't be executed via query service. Has it been resolved, and there are indeed no exceptions?